static/js/components/OrderSummaryCard.js

```diff
@@ -107,6 +152,8 @@ export class OrderSummaryCard extends React.Component<Props> {
           ? this.renderAppliedCoupons()
           : null}
 
+        {this.renderRefunds()}
+
```

Comment:
I think we should add the paid tag here, and make them conditional on each other. Maybe something like this ` {(refunds === null || refunds.length === 0) ? this.renderFulfilledTag() : this.renderRefunds()}`.

Also, this variable doesn't reflect the precise status of the order, it just reflects before payment (cart checkout stage) or after payment( order receipt page)

Reply:
so you will notice now on the refunded page both tags are being displayed

Reply:
Changed this accordingly (see screenshots). 

No refund:
![image](https://user-images.githubusercontent.com/945611/161085934-4caacf67-4843-44e6-8cdf-cf7aaa8f92d5.png)
Refund:
![image](https://user-images.githubusercontent.com/945611/161086014-0a124047-8a76-4928-b649-bcd4c588abc5.png)